e2e/regressions/build.go

```diff
@@ -176,6 +176,17 @@ func (c *regressionsTests) issue4583(t *testing.T) {
 	)
 }
 
+func (c *regressionsTests) issue4943(t *testing.T) {
+	c.env.RunSingularity(
+		t,
+		e2e.WithProfile(e2e.UserProfile),
```

Comment:
Could you use build instead ? As we are in `regressions/build.go`

```
e2e.WithProfile(e2e.UserProfile),
e2e.WithCommand("build"),
e2e.WithArgs("--force", "/dev/null", "docker://gitlab-registry.cern.ch/linuxsupport/cc7-base:20191107"),
e2e.ExpectExit(0),
```

Reply:
OK done.